ui/packages/catalog-ui-search/src/main/webapp/react-component/container/multi-select-actions/multi-select-actions.tsx

```diff
@@ -41,10 +43,13 @@ class MultiSelectActions extends React.Component<Props, State> {
   }
 
   handleExport = () => {
+    const selectedResults = store.getSelectedResults()
```

Comment:
This is another case where we want to avoid pulling in the store since it's used across multiple routes.

```
const selectedResults = this.props.selectionInterface.getSelectedResults()
```

This will work across routes.